### ISSUE TITLE
Add support for 10.0.14393.1737

### ISSUE THREAD

**Issue by koi123** [stascorp/rdpwrap#322]
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/1374765/termsrv.zip)
it suddenly cannot 10.0.14393.1737

**Comment by m0x3** [comment]
+1, same problem after update
windows 10 pro x64
10.0.14393.1737

**Comment by thaguitarist** [comment]
+1, since update kb4041691  rdp wrapper is stating [Not Supported] and doesn't work.
Deleting this update seems to "fix" the issue for now.

**Comment by Alita-B** [comment]
Same problem after tonight's windows update, but Winver report Version 1607 (build SO 14393.1770)

![image](https://user-images.githubusercontent.com/32711327/31449541-77958858-aea7-11e7-8b69-9b2185b4dcb9.png)

**Comment by sinsinpub** [comment]
btw, `rfxvmt.dll` is still there, unchanged. will work if replaces `termsrv.dll` with old version.

**Comment by maaboo** [comment]
Confirm, KB4041691 breaks RDP Wrapper. Reinstallation doesn't help.

**Comment by Curious75** [comment]
Windows 10 Pro version1607 (14393.1770). 
Confirm, update KB4041691 breaks RDP Wrapper:  Listener state [not supported].
Delete this update helps to work normal (as before).

**Comment by johndewood** [comment]
Here is the dll
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/1380068/termsrv.zip)

**Comment by rcg4u** [comment]
latest update KB4041691 has broken rdpwrap  it states not supported, and when you log in trying to shadow it logs the other account out. removing the latest update seems to fix the problem temporary. If you dont want windows to auto download updates, change your connection to metered and windows cannot download an update automatically when its set to metered.

**Comment by thaguitarist** [comment]
Until the lastest update can be supported by rdpwrap, you can uninstall the kb4041691 and use the following Microsoft tool to hide the update so it won't come back unless you "unhide" it. [Wushowhide.diagcab](http://download.microsoft.com/download/f/2/2/f22d5fdb-59cd-4275-8c95-1be17bf70b21/wushowhide.diagcab)

**Comment by muencht** [comment]
I also just installed kb4041691 on 10/12/2017 and RDPWrap no longer worked. It is interesting that in WUP it showed as "2017-10 Cumulative Update for Windows 10 Version 1607 for x64-based Systems (KB4041691)" but when I uninstalled it showed as "Security Update for Microsoft Windows (KB4041691)". I already have the Microsoft Hide/Show utility and made sure to hide the bugger. 

After the removal of  kb4041691 -- RDPWrap 1.6.1 again works. 

It is also interesting to note -- if others are not aware of it -- that M$ can (and will) UNHIDE updates that you have hidden. They especially like to do this for 'Feature' updates such as W10AU (Anniversary Update) and upcoming CU (Creators Update). Thank you MICROSOFT. 

Hopefully the author(s) will fix this so I can re-apply KB4041691. 

To the author(s): RDPWrap is a great product; thank you for your work.
